rustler/src/env.rs

```diff
@@ -23,7 +23,7 @@ pub struct Env<'a> {
 
 /// Two environments are equal if they're the same `NIF_ENV` value.
 ///
-/// A `Env<'a>` is equal to a `Env<'b>` iff `'a` and `'b` are the same lifetime.
+/// A `Env<'a>` is equal to a `Env<'b>` if `'a` and `'b` are the same lifetime.
```

Comment:
Ah thanks for pointing that out. As this is not something that was intuitive to me, I've updated it to "if and only if" to avoid user confusion in the future.